api/spec/requests/spree/api/checkouts_spec.rb

```diff
@@ -82,8 +82,6 @@ module Spree::Api
         end
 
         it "can update addresses and transition from address to delivery" do
-          pending "SQLite adapter fails to provide a unique index" if ActiveRecord::Base.connection.adapter_name == "SQLite"
```

Comment:
I confirmed with the Docker setup that the tests are green now :four_leaf_clover: